### ISSUE TITLE
Overall design and state of project

### ISSUE THREAD

**Issue by zalmoxis** [elnkosc/cryptrade#4]
Hi @elnkosc , I don't know how to contact you so I'm creating this issue. I was reading through the code and I quite like how thorough it is. I have a few things I was wondering about ....
I see you have used design patterns extensively. Are you a C++ or Java programmer in your day job? Did you use any kind of code generator for this project?
How likely is it that the 4 exchanges that you integrated are still compatible with this project. Do you have any intention to continue work on this if I contribute?

I started adding some database  features to record all transactions in order to build some historical analytics and financial position tracking. Before I go any further with this I'd like to get your opinion on the state of the project and interest level.

**Comment by elnkosc** [comment]
Hi Darie,

I haven't checked if they recently still work as I noticed that the
exchange APIs tend to change now and then. Ideally I would need to add
tests to validate compatibility with the exchange APIs. I'm not very
actively developing it at the moment but there are certainly changes I'd
like to make still. Feel free to contribute of course. I used to be a C++
developer long ago but today programming has moved from profession to hobby
and this projeft for me was to get more hands-on with Python as I like to
use it for creating all  kind of simple tools in a quick way.;-)

BR, Koen

Op wo 31 mrt. 2021 om 02:11 schreef Darie Petrov ***@***.***>:
